packages/extension/src/components/pages/Login/Login.tsx

```diff
@@ -154,9 +178,17 @@ export const Login: FC = () => {
           });
       }
     } else {
-      setPasswordError('Incorrect password');
+      if (currentAttempts >= maxAttempts - 1) {
```

Comment:
```suggestion
      if (currentAttempts >= MAX_ATTEMPTS - 1) {
```
because you changed the const above

Reply:
All done, sorry i was focused on presentation